### PR TITLE
Update Migrations.php

### PR DESCRIPTION
**Pull request by free6k**
Added support Postrgresql :(

was error:

ERROR: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "`"
LINE 1: SELECT * FROM `phalcon_migrations` ORDER BY `version` DESC L...

### CODE REVIEW

File: scripts/Phalcon/Migrations.php [hunk 1]
```diff
@@ -516,7 +516,7 @@ public static function getCompletedVersions($options)
         if (isset($options['migrationsInDb']) && (bool)$options['migrationsInDb']) {
             /** @var AdapterInterface $connection */
             $connection = self::$_storage;
-            $completedVersions = $connection->query('SELECT `version` FROM `phalcon_migrations` ORDER BY `version` DESC')->fetchAll();
+            $completedVersions = $connection->query('SELECT version FROM phalcon_migrations ORDER BY version DESC')->fetchAll();
```
Comment (sergeyklay) [inline scripts/Phalcon/Migrations.php]: Best way is to use escape methods from Dialect class

Comment (free6k) [inline scripts/Phalcon/Migrations.php]: Postgresql don't support this escape, oracle don't support. You write all code for mysql, mysql dont support common SQL standarts...

Comment (free6k) [inline scripts/Phalcon/Migrations.php]: Best way for this case it use PDO variables binding, like did it yii2 framework

Comment (sergeyklay) [inline scripts/Phalcon/Migrations.php]: Yeah, this is why I suggest to improve this PR. What do you think about this?

Comment (free6k) [inline scripts/Phalcon/Migrations.php]: I will make

Comment (free6k) [inline scripts/Phalcon/Migrations.php]: ping @sergeyklay, look pull request

Comment (free6k) [inline scripts/Phalcon/Migrations.php]: About ```$connection->query('SELECT version FROM phalcon_migrations ORDER BY version DESC')```, i didnt found method for normal queryBuilder for Phalcon.

write ```$connection->query('SELECT '. $connection->escapeString('version') .' .... ``` not good

Comment (sergeyklay) [inline scripts/Phalcon/Migrations.php]: https://docs.phalconphp.com/en/latest/api/Phalcon_Mvc_Model_Query_Builder.html

Comment (free6k) [inline scripts/Phalcon/Migrations.php]: This query builder working only with model. Here need classical query builder.

```
$qb =  new Builder([
   'from' => self::$migration_log_table,
   'columns' => '*',
   'order' => ['version DESC'],
   'limit' => 1,
]);
echo $qb->getQuery();
```
i have next error:
```
Error: At least one model is required to build the query
```

or you want that i created model for migration table? i think it is wrong

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**